### DISCUSSION TITLE
FAIL: unknown cheatcode with selector 0x73b9aa91

### DISCUSSION THREAD

**Discussion by KiseSaiyajin** [Cyfrin/foundry-full-course-cu#3129] [Q&A] [answered]
I tried to test my RaffleTest.t.sol with forge test command and I got this message

Ran 1 test for test/unit/RaffleTest.t.sol:RaffleTest
[FAIL: unknown cheatcode with selector 0x73b9aa91; you may have a mismatch between the `Vm` interface (likely in `forge-std`) and the `forge` version] setUp() (gas: 0)
Suite result: FAILED. 0 passed; 1 failed; 0 skipped; finished in 8.34ms (0.00ns CPU time)

Ran 1 test suite in 13.63ms (8.34ms CPU time): 0 tests passed, 1 failed, 0 skipped (1 total tests)

Failing tests:
Encountered 1 failing test in test/unit/RaffleTest.t.sol:RaffleTest
[FAIL: unknown cheatcode with selector 0x73b9aa91; you may have a mismatch between the `Vm` interface (likely in `forge-std`) and the `forge` version] setUp() (gas: 0)

I thought that my foundry version was outdated and update foundry to 0.3.0 version, but problem still there.
My setUp function
```
function setUp() external {
        DeployRaffle deployer = new DeployRaffle();
        (raffle, helperConfig) = deployer.deployRaffle();
        vm.deal(PLAYER, STARTING_PLAYER_BALANCE);
        HelperConfig.NetworkConfig memory config = helperConfig.getConfig();
        entrenceFee = config.entrenceFee;
        interval = config.interval;
        vrfCoordinator = config.vrfCoordinator;
        gasLane = config.gasLane;
        callbackGasLimit = config.callbackGasLimit;
        subscriptionId = config.subscriptionId;
        account = config.account;
    }
    ```
    I try to find something similar in internet, but I got nothing. Maybe someone have this mistake?

**Comment by EngrPips** [comment]
Did you import `Vm` in the test file?

**Reply by KiseSaiyajin** [reply]
Yeah, 
```// SPDX-License-Identifier: MIT
pragma solidity 0.8.19;
import {Test, console2} from "forge-std/Test.sol";
import {DeployRaffle} from "script/DeployRaffle.s.sol";
import {Raffle} from "src/Raffle.sol";
import {HelperConfig} from "script/HelperConfig.s.sol";
import {Vm} from "forge-std/Vm.sol";
import {VRFCoordinatorV2_5Mock} from "@chainlink/contracts/src/v0.8/vrf/mocks/VRFCoordinatorV2_5Mock.sol";
import {CodeConstants} from "script/HelperConfig.s.sol";
```
these are all imports

**Reply by EngrPips** [reply, marked as answer]
It seems correct to me, Maybe delete the `forge-std` folder and re-install with `forge install foundry-rs/forge-std` and retry your test. Run `forge clean` and delete the cache folder before retrying and let's see if the issue persists.
